src/software/ai/hl/stp/play/corner_kick_play.cpp

```diff
@@ -15,7 +16,7 @@ using namespace Passing;
 
 const std::string CornerKickPlay::name = "Corner Kick Play";
 
-CornerKickPlay::CornerKickPlay() : MAX_TIME_TO_COMMIT_TO_PASS(Duration::fromSeconds(5)) {}
+CornerKickPlay::CornerKickPlay() : MAX_TIME_TO_COMMIT_TO_PASS(Duration::fromSeconds(3)) {}
```

Comment:
Magic number

Reply:
Changed to dynamic param

Reply:
@jonathanlew can I resolve this discussion?